app/controllers/dashboard_controller.rb

```diff
@@ -64,7 +64,7 @@ def upcoming_events_for_user
   def all_events(workshops)
     course = Course.next
     meeting = Meeting.next
-    event = Event.next
+    event = Event.upcoming(3)
```

Comment:
@deniseyu What do you think about using a constant here ? Something like `NUMBER_OF_EVENTS = 3`.